scripts/compare_all_runs_round3.py

```diff
@@ -0,0 +1,81 @@
+import pandas as pd
+import subprocess
+import argparse
+import os
+import pdftotext
+import re
+
+def create_file_dct(run_file):
+    eval_path = os.path.join(args.anserini_root, 'eval/trec_eval/trec_eval')
+    measure_judged_path = os.path.join(args.anserini_root, 'tools/eval/measure_judged.py ')
+    output = subprocess.check_output(
+        f'{eval_path} -c -m ndcg_cut.10 -m P.5 -m bpref -m map -m rbp.p=0.5 {qrels} {runs}/{run_file}',
+        shell=True)
+    output_lst = output.split()
+
+    J_measure = subprocess.check_output(
+        f'python3 {measure_judged_path}--qrels {qrels} --cutoffs 10 5 1000 --run {runs}/{run_file}', shell=True)
+    J_measure_lst = J_measure.split()
+
+    clean_output = [float(output_lst[11]), float(J_measure_lst[2]), float(output_lst[8]), float(J_measure_lst[5]),
+                    float(output_lst[14]), float(output_lst[5]), float(output_lst[2]), float(J_measure_lst[8])]
+    return clean_output
+
+
+def compare_score(actual_score, row, score_type_lst):
+    run_name = row.run
+    for i in range(len(actual_score)):
+        expected_score = row[i + 4]
+        if actual_score[i] != expected_score:
+            score_type = score_type_lst[i]
+            print(f"In the run {run_name}, the score for {score_type} doesn't match")
```

Comment:
single quote - be consistent.

fwiw - use single quotes throughout our code base.